@xen-orchestra/web-core/lib/components/link/VtsLink.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <component :is="component" :class="classes" class="vts-link" v-bind="attributes">
+    <UiIcon :icon="icon" color="current" />
```

Comment:
Since the icon component has been updated to `VtsIcon`, what do you think of rebasing this PR and using the new version of the icon component?
```suggestion
    <VtsIcon :icon color="current" />
```